src/luv.c

```diff
@@ -242,6 +242,13 @@ static const luaL_Reg luv_functions[] = {
   {"version", luv_version},
   {"version_string", luv_version_string},
 
+  // thread.c
+  {"new_thread", luv_new_thread},
+  {"thread_create", luv_thread_create}, 
+  {"thread_equal", luv_thread_equal},
+  {"thread_self", luv_thread_self},
+  {"thread_join", luv_thread_join},
+
```

Comment:
New only create handle,and create real start thread,please don't cost more time on it,because will  change lot in afew days.